### ISSUE TITLE
Systemd cgroups don't work

### ISSUE THREAD

**Issue by LK4D4** [docker/docker#19140]
```
docker run -it --rm busybox
docker: Error response from daemon: Cannot start container 7de93d77778179628252cc4fd58b8559ed675fc7a8e1f448a888acf3179e1154: [9] System error: Invalid unit name '/docker'.
```
I have systemd-228 and kernel 4.3.3

**Comment by thaJeztah** [comment]
Related to https://github.com/docker/docker/issues/17126 ?

**Comment by LK4D4** [comment]
@thaJeztah Looks like that

**Comment by hqhq** [comment]
It's not really related to #17126 , but they are all cgroup-parent problems on systemd cgroup.
Error in #17126 won't happen by default now because we are using fs cgroup by default, but it will still error out on systemd cgroup. So I opened a PR #19154 to fix #17126 .
